content/fr/articles/js/react/react-native-introduction/index.md

```diff
@@ -87,3 +86,46 @@ AppRegistry.registerComponent('PutainDeBiere', () => App);
   <img src="welcome.png" alt="welcome PutainDeBiere preview" />
   <figcaption>Une bien jolie première étape</figcaption>
 </figure>
+
+## Récupération des données
+
+Afin de requêter notre API, React Native nous offre plusieurs plusieurs solutions: `fetch()` ou `XMLHttpRequest`. Tenez vous en **uniquement** à l'utilisation de la première, la deuxième n'étant présente que pour assurer une compatibilité avec des librairies tierces.
+
+Ainsi, nous allons modifier notre Component `<App>` afin de faire une requête simple de toutes les bières avant le montage de celui-ci.
+
+*/!\ Notre clé API doit être encodé en base64. La function `btoa()` n'étant pas disponible en React Native, il est nécessaire d'installer une dépendance.*
+
+```bash
+npm install --save base-64
+```
+
+```javascript
+import base64 from 'base-64'; // ajoutez l'import de la dépendance
+
+class App extends Component {
+  componentWillMount() {
+    const username = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx' // changez pour votre clé API
```

Comment:
React Native does not handle them. So it's a choice over using a babel plugin like "transform-inline-environment-variables" (and rewrite / override the default babel config) or a dotenv like alternative to simplify the tutorial. I should mention it.